### PR TITLE
Inline code documentation

### PR DESCRIPTION
**Pull request by kari0003**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,325 @@
-# swift-sdk
-High level SDK for Swift
+# Paylike's Swift SDK
+
```
Comment (kocsislaci) [inline README.md:2]: Should we add a CI badge here?
Like in the other swift packages :)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,325 @@
...
+This library includes the high level components providing complete payment forms to use the Paylike ecosystem.  
+If you are looking for our core elements required to implement payment flow towards the API, [check here](https://github.com/paylike/swift-engine).
+
+## Table of Contents
+---
+- [Paylike's Swift SDK](#paylikes-swift-sdk)
+  - [Table of Contents](#table-of-contents)
+  - [General Info](#general-info)
+    - [Purpose of this library](#purpose-of-this-library)
+    - [# Used Technologies, and iOS target](#used-technologies-and-ios-target)
```
Comment (kocsislaci) [inline README.md]: Is this # intended here?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,325 @@
...
+        .package(url: "https://github.com/paylike/swift-sdk", .upToNextMajor(from: "0.1.0")),
+        // ...
+    ],
+    // ...
+)
+```
+
+## TL;DR
+---
+Don't forget to create your **OWN** API key (merchant ID) for the Paylike ecosystem. See: https://paylike.io/platforms/integration & https://github.com/paylike/api-docs#getting-an-api-key
```
Comment (kocsislaci) [inline README.md]: Should we add some custom text for the links? 
Like: "Integration Docs & Getting an API key"

**Review by kocsislaci — COMMENTED** [review]
I left some comments on the readme :)

File: README.md (hunk 1 above)
Comment (kari0003) [reply README.md:2]: I did initially, but as it was pointing at the other project, it was broken :D

**Pull request merged**